ang/civicase/case/details/directives/tab-content/details.html

```diff
@@ -0,0 +1,8 @@
+<div
+  ng-if="isMainContentVisible()"
```

Comment:
Dont we need placeholder for this?

Reply:
good catch @deb1990 , unfortunately since the tab is added dynamically AFTER the case is loaded, there is no way for us to know if the tab is visible. This is how it looks when it's loading:

![Manage_Cases_case8004 (8)](https://user-images.githubusercontent.com/1642119/84203306-3e396380-aa77-11ea-8460-fe86f21e08bf.png)

The solution would be to do the reverse: to always append the tab and remove it when no tabs custom groups are found. Given the amount of time already spend on this, the importance of this placeholder feature, and the list of remaining tasks, I'd say we should skip this one.